roles/container_runtime/tasks/extra_storage_setup.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Create file system on extra volume device
+  filesystem:
+    fstype: "{{ item.filesystem }}"
+    dev: "{{ item.device }}"
+    force: "{{ item.force|default(omit) }}"
+  with_items: "{{ container_runtime_extra_storage|default([]) }}"
```

Comment:
Add the default for container_runtime_extra_storage| to role's defaults/main.yml.